src/HeapDump/Program.cs

```diff
@@ -174,6 +183,23 @@ private static int MainWorker(string[] args)
                         }
                         Console.WriteLine("Generation To Trigger: " + dumper.GenerationToTrigger);
                     }
+                    else if (arg.StartsWith("/SymbolsAuth:", StringComparison.OrdinalIgnoreCase))
+                    {
+                        string authTypesStr = arg.Substring(13);
+                        if (TryParseSymbolsAuthenticationTypes(authTypesStr, out var parsedAuthTypes))
```

Comment:
Why not pass symbolsAuth rather than introducting a new local? In all continuing cases, symbolsAuth is immediately overwritten with this value anyway.